droidlet/lowlevel/minecraft/tests/test_cuberite_agent_connection.py

```diff
@@ -0,0 +1,37 @@
+"""
+Copyright (c) Facebook, Inc. and its affiliates.
+"""
+import os
+import signal
+import subprocess
+import time
+import unittest
+from agents.craftassist.craftassist_agent import CraftAssistAgent
```

Comment:
I think this test should go under `agents/craftassist/tests/` instead of `lowlevel` because this tests only our current agent with cuberite. Doing so will also preserve what's proposed [here](https://github.com/facebookresearch/droidlet/issues/337) so we don't import `agent` in any of the subfolders going forward.

Reply:
done